src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/CustomObject.txt

```diff
@@ -1,23 +1,91 @@
-﻿>> Value(Index(ParseJson("[5]"), 1))
+﻿// Value tests
+>> Value(Index(ParseJson("[5]"), 1))
 5
 
 >> Value(ParseJson("5"))
 5
 
+>> Value(ParseJson("true"))
+1
+
+>> Value(ParseJson("false"))
+0
+
+>> Value(ParseJson("""5"""))
+5
+
+>> Value(ParseJson("""a"""))
+#Error
+
+>> Value(ParseJson("null"))
+Blank()
+
+>> Value(ParseJson("{""a"": 5}").a)
```

Comment:
>.a) [](http://example.com/codeflow?start=32&length=3)

Can you add Case sensitivity tests?
Json is case sensitive  :| ...

ParseJson("{a :15, A : 20}").a // 15
ParseJson("{a :15, A : 20}").A // 20

Reply:
Done